### ISSUE TITLE
[BUG]: Help click video and click back and get kicked out off app

### ISSUE THREAD

**Issue by Blackley7** [yuliskov/SmartTube#3850]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

Latest version

### Device Type

Smart TV/Box

### Affected Android

Android box

### Steps to reproduce the bug

Hi is anyone else getting when they go on to a youtube video and the click back to come out the video it kicks you out the app and then you go back in on another video and it does the same ??

### Actual behavior

Hi is anyone else getting when they go on to a youtube video and the click back to come out the video it kicks you out the app and then you go back in on another video and it does the same ??

### Additional information

Amazon fire tv box

**Comment by zanzibar707** [comment]
yup it's happening to me as well  on my fire stick an I have updated twice really takes the enjoyment on using this app right now
